content/docs/new

```diff
@@ -0,0 +1 @@
+
```

Comment:
Could you rename the `content/docs/new` file to `content/docs/_index.md` and add the following copy:

```
---
title: "Docs Index"
description: "index of the docs section"
date: 2020-10-22
toc: true
---

This is index content
```